FizzBuzz - single parameter.rb

```diff
@@ -0,0 +1,47 @@
+def fizzBuzz(isFB)
+
+  def printFizzBuzz(isFB)
+    if
+      isFB%3 == 0 && isFB%5 == 0
+      puts "FizzBuzz"
+    elsif isFB%3 == 0
+      puts "Fizz"
+    elsif isFB%5 == 0
+      puts "Buzz"
+    else
+      puts "No FizzBuzz for you, sport!"
+    end
+  end
+
+  def isNumber(isNum)
+    isNum = isNum.to_s
+    /\A[+-]?\d+(\.[\d]+)?\z/.match isNum
+  end
+
+  if isNumber(isFB)
+    isFB = isFB.to_f
+    if isFB%1 == 0
+    else
+      puts "Would you like to round your number #{isFB}? Y/N"
+      q = gets.chomp
```

Comment:
you can call `downcase` after the `gets.chomp` to always have a lower-case letter as an answer. Given that you will need only `if q == 'y'` to proceed further

Reply:
Must've skipped my mind, because in the redesigned LeftPad and FizzBuzz v2 I've used it every time. Thanks for pointing that out.